src/include/daos_srv/container.h

```diff
@@ -120,6 +125,23 @@ struct ds_cont_child {
 	uint32_t		 sc_dtx_resync_ver;
 };
 
+typedef uint64_t (*agg_param_get_eph_t)(struct ds_cont_child *cont);
+struct agg_param {
+	void			*ap_data;
+	struct ds_cont_child	*ap_cont;
+	daos_epoch_t		*ap_full_scan_hlc;
```

Comment:
Why it's a pointer?

Reply:
same reason, because sharing cont_child_aggregate().